### PR TITLE
B2B-Sync remove Preview tag

### PR DESCRIPTION
**Pull request by BillSproule**
Updated title on page to remove "(Preview)".  Also updated availability notice to only refer to Enterprise ring.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/b2b-sync.md [hunk 1]
```diff
@@ -18,14 +18,12 @@ search.appverid:
...
 
 > [!IMPORTANT]
-> This feature is currently enabled in the Insiders ring only (build 19.012.0121.0005 or later - [see the release notes for rollout info](https://support.office.com/article/845dcf18-f921-435e-bf28-4e24b95e5fc0)). To try this feature, join the [Windows Insider program](https://insider.windows.com/) or the [Office Insider](https://products.office.com/office-insider) program. <br>Alternatively, run the following command to have only the sync client join the Insider program:<br>
-PC:	reg.exe add HKCU\Software\Microsoft\OneDrive /v EnableTeamTier_Internal /t REG_DWORD /d 1 /f <br>Mac: defaults write com.microsoft.OneDrive Tier "Insiders"
-> This feature is expected to be enabled for most customers on or after June 17, 2019.  Customers who have opted into the Enterprise audience will see the feature enabled when build 19.086.* or later is available to them.
+> This feature is not yet enabled for our customers in our Enterprise ring. Once build 19.086.* or newer is available to Enterprise ring users, the B2B Sync feature will be enabled.  [See the release notes for rollout info](https://support.office.com/article/845dcf18-f921-435e-bf28-4e24b95e5fc0).
```
Comment (JohanFreelancer9) [inline OneDrive/b2b-sync.md]:
```suggestion
> This feature is not yet enabled for our customers in our Enterprise ring. Once build 19.086.* or newer is available to Enterprise ring users, the B2B Sync feature will be enabled. [See the release notes for rollout info](https://support.office.com/article/845dcf18-f921-435e-bf28-4e24b95e5fc0).
```

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by JohanFreelancer9** [comment]
Copy review is done.

**Pull request merged**